udata_front/theme/gouvfr/templates/dataset/resource/card.html

```diff
@@ -1,7 +1,7 @@
 {% set resource_format = resource.format|trim|lower %}
 {% set expanded = expanded or "false" %}
 <article
-    class="card resource-card {% if dark %}dark{% endif %} {% if resource.from_community %}resource-card-community{% endif %}">
+    class="card resource-card {% if resource.from_community %}resource-card-community{% endif %}">
```

Comment:
Could we change the colors of the resource owner in case of community card.
It's not the only contrast issue, but let's not add more ;)
See:
![image](https://user-images.githubusercontent.com/28541902/136770869-7f88b84f-4ba2-4a75-b27f-af81faa136e1.png)